integration_test/test_videoroom/mix.lock

```diff
@@ -1,6 +1,6 @@
 %{
```

Comment:
please cleanup mix.lock from unused deps

Reply:
That means that CI is out of date, this should fail on lint 🤔